website/content/docs/secrets/pki/troubleshooting-acme.mdx

```diff
@@ -0,0 +1,198 @@
+---
+layout: docs
+page_title: 'PKI - Secrets Engine: Troubleshooting ACME'
+description: The PKI secrets engine for Vault generates TLS certificates.
+---
+
+# PKI Secrets Engine - Troubleshooting ACME
+
+Below are suggestions for troubleshooting issues with the ACME clients
+requesting access from Vault. Different clients will behave in different
+ways and have different ways of viewing more details about failures.
+
+## Contacting support
+
+When contacting Hashicorp Support with ACME related issues or filing a
+GitHub issue, the following pieces of information are helpful for Support
+and Engineering to investigate and reproduce any issues related to ACME,
+in addition to the regularly asked pieces of information:
+
+ - **ACME client name and version**; different clients behave differently
+   and certain ACME clients might have bugs or other shortcomings at
+   particular versions.
+ - **ACME client logs**, if any; on failure, `certbot` will give the location
+   of debug logs it has produced while attempting to request certificates.
+   This frequently contains helpful information about requests to the Vault
+   cluster and the server's response. Other clients may or may not behave
+   similarly; in the event of not having log information, full output in
+   verbose (or debug mode) of the client would be helpful.
+ - **Vault server DEBUG level logs**; many ACME errors are hidden from the
+   client as they contain sensitive information, but are instead logged,
+   when appropriate, to the server logs at a DEBUG level.
+
+## Debugging cluster path issues
+
+If ACME works on some nodes of a Vault Enterprise cluster but not on
+others, likely it means that the cluster address has not been set.
```

Comment:
```suggestion
```

I suggest moving the intro stuff to a "Cause" section after "Symptoms" so it is easier to see when scanning.